pkg/kinds/dashboard/devenv_test.go

```diff
@@ -0,0 +1,179 @@
+package dashboard_test
+
+import (
+	"encoding/json"
+	"io"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"sort"
+	"testing"
+
+	"cuelang.org/go/cue/errors"
+	"github.com/grafana/kindsys/encoding"
+	"github.com/stretchr/testify/require"
+
+	"github.com/grafana/grafana/pkg/kinds"
+	"github.com/grafana/grafana/pkg/kinds/dashboard"
+	"github.com/grafana/grafana/pkg/registry/corekind"
+)
+
+// TODO these should all go away
+var skiplist = map[string]string{
```

Comment:
as noted in the PR description, it's out of scope for this PR to actually fix most of these issues, but i've aimed to write the test and give descriptions in this skiplist such that it's easy for us to burn these down later.